src/main/scala/circt/Intrinisic.scala

```diff
@@ -0,0 +1,15 @@
+
+// SPDX-License-Identifier: Apache-2.0
+
+/** This has to be in a separate file and package to generate the correct class
+  *  in the annotation.
+  */
+
+package circt
+
+import firrtl.annotations.SingleTargetAnnotation
+import firrtl.annotations.ModuleTarget
+
+case class intrinsic(target: ModuleTarget, intrinsic: String) extends SingleTargetAnnotation[ModuleTarget] {
```

Comment:
```suggestion
case class Intrinsic(target: ModuleTarget, intrinsic: String) extends SingleTargetAnnotation[ModuleTarget] {
```

Is there a reason to deviate from Scala style here to make this lowercase?

Reply:
For the not great reason that the class name appears in the output (there doesn't seem to be a way to make the annotation json have "class=whatever_you_want_which_isnt_based_on_scala_name").

Probably easy to fix.